common/_util.h

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019 Datadog, Inc.
+#ifndef DATADOG_AGENT_SIX_UTIL__H
+#define DATADOG_AGENT_SIX_UTIL__H
+#include <Python.h>
+#include <six_types.h>
+
+#ifdef DATADOG_AGENT_THREE
+PyMODINIT_FUNC PyInit__util(void);
+#    define CStringFromPyString(x) PyUnicode_FromString(x)
```

Comment:
Both define of `CStringFromPyString` could be moved to sixstrings to be used everywhere.
Also since we converting char* to PyObject should we not rename this `PyStringFromCString` ?